cmd/skaffold/app/cmd/run.go

```diff
@@ -54,7 +54,7 @@ func doRun(ctx context.Context, out io.Writer) error {
 			}
 		}
 
-		// Render
+		// Manifests
```

Comment:
this should probably be kept the same or just removed as the comment doesn't add anything really

Reply:
done (in next version of PR)